SonarQube.Client/Api/V7_20/GetIssuesRequest.cs

```diff
@@ -145,6 +150,12 @@ private class ServerIssue
             [JsonProperty("textRange")]
             public ServerIssueTextRange TextRange { get; set; }
 
+            [JsonProperty("creationDate")]
+            public virtual DateTimeOffset CreationDate { get; set; }
```

Comment:
The web service returns data in a format that is handled correctly by the JSON deserializer without us having to do anything special.

This library does have a special `JavaDateConverter` class. However, it looks like this exists to work round a bug when sending data to `GetNotificationsRequest`. The date parameter sent to that API is encoded in a special way by the `JavaDateConverter`. There is no special handling of the date values returned by that API.